### PR TITLE
Adding BVT as a GAC performance variable

### PR DESCRIPTION
**Pull request by hunterbarber**
## Fixes/Resolves:
Adds BVT as a performance variable into the GAC model

## Summary/Motivation:
BVT is a useful metric for validation, all intermediate variables were already present, and it just required adding the var and constraint

## Changes proposed in this PR:
- Add BVT into GAC unit model
- Additions reflected in documentation

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: watertap/unit_models/gac.py [hunk 1]
```diff
@@ -546,6 +546,14 @@ def eq_isobaric_adsorbed_contam(b, t):
             doc="Elapsed time between GAC replacement in adsorber bed in operation",
         )
 
+        self.bvt = Var(
```
Comment (adam-a-a) [inline watertap/unit_models/gac.py]:
```suggestion
        self.bed_volumes_treated = Var(
```
We generally try to be more explicit about naming (even though bvt is pretty convenient).

File: docs/technical_reference/unit_models/gac.rst [hunk 2]
```diff
@@ -131,6 +131,7 @@ variables in the model.
    "Mass throughput ratio", ":math:`T`", "mass_throughput", "None", ":math:`\text{dimensionless}`"
    "Residence time", ":math:`\tau`", "res_time", "None", ":math:`\text{s}`"
    "Elapsed operation time", ":math:`t_{op}`", "elap_time", "None", ":math:`\text{s}`"
+   "Bed volumes treated", ":math:`BVT`", "bvt", "None", ":math:`\text{dimensionless}`"
```
Comment (adam-a-a) [inline docs/technical_reference/unit_models/gac.rst]:
```suggestion
   "Bed volumes treated", ":math:`BVT`", "bed_volumes_treated", "None", ":math:`\text{dimensionless}`"
```

File: watertap/unit_models/gac.py [hunk 3]
```diff
@@ -851,6 +859,10 @@ def eq_velocity_relation(b):
         def eq_length_bed(b):
             return b.bed_length == b.velocity_sup * b.ebct
 
+        @self.Constraint(doc="Bed volumes treated")
+        def eq_bvt(b):
+            return b.bvt * b.res_time == b.elap_time * b.bed_voidage
```
Comment (adam-a-a) [inline watertap/unit_models/gac.py]:
```suggestion
            return b.bed_volumes_treated * b.res_time == b.elap_time * b.bed_voidage
```

Comment (adam-a-a) [reply watertap/unit_models/gac.py]: And modify across the board

**Review by adam-a-a — CHANGES_REQUESTED** [review]
LGTM- just suggesting to expand the name before we merge.

**Review by agarciadiego — APPROVED** [review]
LGTM

**Review by kurbansitterley — APPROVED** [review]
Looks straightforward to me.

File: watertap/unit_models/gac.py [hunk 4]
```diff
@@ -1359,6 +1371,9 @@ def calculate_scaling_factors(self):
         if iscale.get_scaling_factor(self.elap_time) is None:
             iscale.set_scaling_factor(self.elap_time, 1e-6)
 
+        if iscale.get_scaling_factor(self.bvt) is None:
```
Comment (hunterbarber) [inline watertap/unit_models/gac.py]:
```suggestion
        if iscale.get_scaling_factor(self.bed_volumes_treated) is None:
```

File: watertap/unit_models/gac.py [hunk 5]
```diff
@@ -1359,6 +1371,9 @@ def calculate_scaling_factors(self):
...
+            iscale.set_scaling_factor(self.bvt, 1e-5)
```
Comment (hunterbarber) [inline watertap/unit_models/gac.py]:
```suggestion
            iscale.set_scaling_factor(self.bed_volumes_treated, 1e-5)
```

File: watertap/unit_models/tests/test_gac.py [hunk 6]
```diff
@@ -395,6 +395,7 @@ def test_solution_robust(self, gac_frame_robust):
         assert pytest.approx(1.139, rel=1e-3) == value(mr.fs.unit.mass_throughput)
         assert pytest.approx(12830000, rel=1e-3) == value(mr.fs.unit.elap_time)
         assert pytest.approx(10.68, rel=1e-3) == value(mr.fs.unit.bed_area)
+        assert pytest.approx(21390, rel=1e-3) == value(mr.fs.unit.bvt)
```
Comment (hunterbarber) [inline watertap/unit_models/tests/test_gac.py]:
```suggestion
        assert pytest.approx(21390, rel=1e-3) == value(mr.fs.unit.bed_volumes_treated)
```

File: watertap/unit_models/gac.py [hunk 7]
```diff
@@ -851,6 +859,10 @@ def eq_velocity_relation(b):
         def eq_length_bed(b):
             return b.bed_length == b.velocity_sup * b.ebct
 
+        @self.Constraint(doc="Bed volumes treated")
+        def eq_bvt(b):
```
Comment (hunterbarber) [inline watertap/unit_models/gac.py]:
```suggestion
        def eq_bed_volumes_treated(b):
```

**Comment by hunterbarber** [comment]
> LGTM

@agarciadiego Sorry, double-clicked re-requesting Adam and didn't mean it for you, you can ignore it!

**Review by adam-a-a — APPROVED** [review]

**Pull request merged**